packages/core/src/utils/types.ts

```diff
@@ -11,7 +11,7 @@ export interface BaseElementEvents {
   /**
    * Dispatched when an error is encountered
    */
-  error: Error
+  error: Error | any
```

Comment:
The DX of this seems bad to me. It _might_ be an error, or might not be? Can we at least create an interface `ErrorLike` or something that has a `.message` property on it?